config/locales/helpers/fr.yml

```diff
@@ -4,231 +4,230 @@ fr:
+          Nous apprécions votre aide car nous nous efforçons de favoriser un environnement positif et inclusif pour tous !
+          L'équipe %{community}
+        subject: "Reporter %{community}"
       affected:
         body: |
-          Hi there,
+          Bonjour,
 
-          We noticed some comments (made by others) on your post that violate the %{community} Code of Conduct and/or our Terms of Use. We have zero tolerance for such behavior and are
-          taking appropriate action.
+          Nous avons remarqué que certains commentaires (faits par d'autres personnes) sur votre message violent le %{community} Code de conduite et/ou nos conditions d'utilisation. Nous avons une tolérance zéro pour de tels comportements et nous
+          et nous prenons les mesures appropriées.
 
-          Thanks for being awesome, and please don't hesitate to email us with any questions! We welcome all feedback and ideas as we continue working to foster an open and inclusive community.
+          Merci d'avoir été génial, et n'hésitez pas à nous envoyer un e-mail pour toute question ! Tous les commentaires et idées sont les bienvenus pour continuer à favoriser une communauté ouverte et inclusive.
 
-          %{community} Team
-        subject: Courtesy Notice from %{community}
+          L'équipe %{community}
+        subject: Avis de courtoisie de %{communauté}
     listing_helper:
       option:
-        one: "%{cl_name} (%{count} Credit)"
-        other: "%{cl_name} (%{count} Credits)"
+        one: "%{cl_name} (%{count} Crédit)"
+        other: "%{cl_name} (%{count} Crédits)"
     organization_helper:
       option: "%{org_name} (%{unspent})"
     rate_limit_checker_helper:
       day:
-        one: day
-        other: days
-      general: How many %{thing} can a new member (%{timeframe} or less) create within any 5 minute period?
+        one: jour
+        other: jours
+      general: Combien de %{thing} un nouveau membre (%{timeframe} ou moins) peut-il créer sur une période de 5 minutes ?
       published:
-        title: Limit number of posts created
-        description: How many posts can someone create within any 30 second period?
+        title: Limiter le nombre de postes créés
+        description: Combien de messages peut-on créer sur une période de 30 secondes ?
       antispam:
-        title: Limit number of posts created by a new member
+        title: Limiter le nombre de postes créés par un nouveau membre
       update:
-        title: Limit number of updates to a post
-        description: How many updates can someone make within any 30 second period? Update API docs when changed.
+        title: Limiter le nombre de mises à jour d'un article
+        description: Combien de mises à jour peut-on faire sur une période de 30 secondes ? Mettre à jour la documentation de l'API en cas de modification.
```

Comment:
oh, interesting! Since the [API docs](https://developers.forem.com/api) are for Forem generally, this hint to the administrator might not make any more sense for a site-configurable setting.

unrelated to your PR's changes, but since I'm reading lots of messages we show users in some context I'm taking notes of things that we might want to revisit.